docs/src/pages/components/data-grid/getting-started/migration-v4.md

```diff
@@ -0,0 +1,432 @@
+---
+title: Data Grid - Migration from v4 to v5
+---
+
+<p class="description">MUI X v5 is now stable!</p>
+
+## Introduction
+
+This a reference for upgrading your site from MUI X v4 to v5. MUI X v5 is fully compatible with MUI Core v5 and can be used with MUI Core v4 with some additional steps.
+Most breaking changes are renaming of CSS classes or variables to improve the consistency of the grid.
+
+## Using MUI X v5 with MUI Core v4
+
+We strongly recommend you to migrate both MUI X and MUI Core to v5.
+Depending on the complexity of the application however, this might not be possible.
+An alternative is to install MUI Core v4 alongside to v5, and to configure them to avoid conflicts.
+This can be achieved with the following steps:
+
+1. First, make sure you have MUI Core v5 installed. If not, install it with these [instructions](/components/data-grid/getting-started/#installation).
```

Comment:
That's confusing, the sentence is talking about MUI core, the instructions are about MUI X. 

```suggestion
1. First, make sure you have MUI X v5 installed. If not, install it with these [instructions](/components/data-grid/getting-started/#installation).
```

Reply:
This was already online, this whole paragraph of Core v4 + X v5 is a copy paste :/
Fixed